.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  tests:
+    name: Test
```

Comment:
I think this has to be named `test` otherwise we end up with a branch protection rule that requires `test` to pass, whereas actually it's `Test` that ends up passing.

https://github.com/alphagov/govuk-saas-config/blob/aebb18d9cd32664c94d2bb63d832804ffec222b8/github/lib/configure_repo.rb#L104

Reply:
https://github.com/alphagov/smart-answers/pull/4692 should fix this.